modules/oadp-configuring-dpa-multiple-bsl.adoc

```diff
@@ -7,7 +7,19 @@
 [id="oadp-configuring-dpa-multiple-bsl_{context}"]
 = Configuring the DPA with more than one BSL
 
-You can configure the DPA with more than one BSL and specify the credentials provided by the cloud provider.  
+You can configure the `DataProtectionApplication` (DPA) custom resource (CR) with more than one `BackupStorageLocation` (BSL) and specify the credentials provided by the cloud provider.
```

Comment:
```suggestion
You can configure the `DataProtectionApplication` (DPA) custom resource (CR) with more than one `BackupStorageLocation` (BSL) CR and specify the credentials provided by the cloud provider.
```

Object name is needed: [guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references)